_posts/2018-01-30-foreman-community-newsletter---january-2018.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Foreman Community Newsletter - January 2018
+date: 2018-01-31 14:35:00
+author: Ori Rabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Events, demos, and the usual round-up
+---
+
+Events heading our way!
+
+### Upcoming Events
+
+* Community Demo - 15th February (URL to be announced)
+
+Time again for the latest round-up of action from the community, covering core,
+proxies, plugins, and more.  As always, you're welcome to participate in the
+show, either via YouTube Live chat, or via our IRC channel
+(Freenode/#theforeman).
+
+* [UX review - Red Hat Repositories page - 31st January](https://www.youtube.com/watch?v=9o1X_Th5Tus)
+This demo will provided a depth look at the new Red Hat Repositories page.
+A page that has been rewritten using react. It adds search and filtering capabilities
+and brings that page inline with patternfly best practices.
+
+#### Upcoming Conferences
+
+* [FOSDEM](https://fosdem.org/2018/): Come see us at our joint stand with oVirt
+on level 1 of building K in Brussels 3-4 Feb
+* [Config Management Camp](http://cfgmgmtcamp.eu/): Join us at the Foreman room (B.2.015) and booth
+in Gent 5-7 Feb. The 7th of Feb is an extra day for everything Foreman related - meet the
+developers, joint developing/debugging sessions and more, [register][hack_day] now!
+
+#### Community Demos / Deep Dives
+
+* [Community Demo - 25th Jan](https://www.youtube.com/watch?v=ekf8Y082HoQ)
+
+In this community demo we saw RSS notifications, fact filtering, a new plugin and tool and more.
+We also updated where to find us in upcoming conferences.
+
+* [Community Demo - 4th Jan](https://www.youtube.com/watch?v=BVTiwSEMcyw)
+
+In this community demo we saw RSS notifications, fact filtering, a new plugin and tool and more.
+We also updated where to find us in upcoming conferences.
```

Comment:
Is having the same text here and on the line 42 intentional?

Reply:
nope, this was supposed to be updated :)